scripts/tpu-cleanup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -x
+set -u
+
+zones=("us-central1-b" "us-central2-b" "europe-west4-a")
+
+for zone in ${zones[@]}; do
+  region=${zone%-*}
+  cluster=xl-ml-test-$region
+  gcloud container clusters get-credentials $cluster --region $region --project xl-ml-test
+
+  # Don't delete any TPUs for pods that are currently running
+  pods_to_save=$(mktemp)
```

Comment:
I wonder what `$(mktemp)` is

Reply:
I think it makes a temp file and write to-be-deleted TPUs there.